vm-setup/roles/v1aX_integration_test/templates/workers_ubuntu.yaml

```diff
@@ -89,14 +89,15 @@ spec:
         - curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
         - echo 'deb https://apt.kubernetes.io/ kubernetes-xenial main' > /etc/apt/sources.list.d/kubernetes.list
         - apt update -y
+        - apt-get install -y conntrack socat
         - apt install docker-ce docker-ce-cli containerd.io -y
         - echo  "Installing kubernetes binaries"
         - curl -L --remote-name-all https://storage.googleapis.com/kubernetes-release/release/{{KUBERNETES_BINARIES_VERSION}}/bin/linux/amd64/{kubeadm,kubelet,kubectl}
         - chmod a+x kubeadm kubelet kubectl
         - mv kubeadm kubelet kubectl /usr/local/bin/
         - mkdir -p /etc/systemd/system/kubelet.service.d
-        - curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/{{KUBERNETES_BINARIES_VERSION}}/build/debs/kubelet.service" | sed "s:/usr/bin:/usr/local/bin:g" > /etc/systemd/system/kubelet.service
-        - curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/{{KUBERNETES_BINARIES_VERSION}}/build/debs/10-kubeadm.conf" | sed "s:/usr/bin:/usr/local/bin:g" > /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+        - curl -sSL "https://raw.githubusercontent.com/kubernetes/release/v0.2.7/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service" | sed "s:/usr/bin:/usr/local/bin:g" > /etc/systemd/system/kubelet.service
```

Comment:
wasn't the whole idea of this was to parameterize any versions?

Reply:
That version is fixed for all versions. 
And, it is a systemd configuration file.

Reply:
I think you are right. Parameterize it as well.

Reply:
replacement done